addons/briefing/XEH_postInit.sqf

```diff
@@ -11,12 +11,20 @@ if (hasInterface) then {
         [_unit] call FUNC(addIntentToZeus);
     }] call CBA_fnc_addEventHandler;
 
+    // TODO: BWC, remove this a while after (can't use movedToSQF = 1 here)
```

Comment:
I don't think we need this at all, beccause ace will auto conver the ace_settings from the description.ext to new cba settings
So we should just get BWC for free

Reply:
I tried that with safe start and it was a no-go. I can give it another go, but not only did it not set the setting correctly, it overwrote the setting to junk.